Projects/TEAM-3/Mykhailenko/QA2022_04_402/ex402/Pages/RegisterPage.cs

```diff
@@ -0,0 +1,44 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using ex402.Utils;
+
+namespace ex402.Pages
+{
+    internal class RegisterPage
+    {
+        public string RegisterUser(string _url, string _fname, string _lname, string _email, string _phone, string _pass)
+        {
+            Values v = new Values();
+            v.driver.Url = _url;
+            IWebElement my_acc = v.driver.FindElement(By.ClassName("dropdown"));
+            my_acc.Click();
+            IWebElement register = v.driver.FindElement(By.LinkText("Register"));
```

Comment:
It is better to init IWebElements, not like variables in the method but like class fields. Because using the current approach you will not be able to reuse your IWebElements in other methods

Reply:
Thanks.
Didn't think about it